{"failedTests":[{"testName":"parallel/test-async-context-frame","timestamp":1725597395142,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win10-COMPILED_BY-vs2022 #30011","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win10-COMPILED_BY-vs2022/30011/","buildNumber":30011,"builtOn":"test-azure_msft-win10_vs2019-x64-2","nodeVersion":"23.0.0","commitHash":"86d6382bf65e33444f03606f21115d20b0ac881a","callStack":[{"upstreamBuild":65590,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":74037,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":62031,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54748"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":89,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (4649.0287ms)\n  ✔ async-hooks\\test-async-local-storage-async-await.js (5034.1818ms)\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (4915.6694ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (5307.1095ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (5150.4393ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (4687.6503ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (4829.2157ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (4795.3637ms)\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (4983.9251ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (5292.808ms)\n  ✖ async-hooks\\test-async-local-storage-misc-stores.js (4864.2643ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-misc-stores.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 10)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-nested.js (5091.4295ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (5440.6416ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (4785.1569ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (5360.7201ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (4773.5844ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (5361.2804ms)\n  ✔ parallel\\test-async-local-storage-bind.js (4565.6877ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (4887.637ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (4881.169ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (4462.2866ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (5318.579ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (5248.6633ms)\n▶ AsyncContextFrame (5519.1971ms)\nℹ tests 23\nℹ suites 1\nℹ pass 22\nℹ fail 1\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 5526.823\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-misc-stores.js (4864.2643ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-misc-stores.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///c:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 10)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":5991.017,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}},{"testName":"parallel/test-async-context-frame","timestamp":1725597395142,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2019-COMPILED_BY-vs2022 #30011","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2019-COMPILED_BY-vs2022/30011/","buildNumber":30011,"builtOn":"test-rackspace-win2019_vs2019-x64-4","nodeVersion":"23.0.0","commitHash":"86d6382bf65e33444f03606f21115d20b0ac881a","callStack":[{"upstreamBuild":65590,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":74037,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":62031,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54748"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":110,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (5232.3498ms)\n  ✔ async-hooks\\test-async-local-storage-async-await.js (4768.9632ms)\n  ✖ async-hooks\\test-async-local-storage-async-functions.js (4540.0772ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-async-functions.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 2)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-dgram.js (5025.3107ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (5314.6788ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (4290.4876ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (5302.9305ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (5491.9166ms)\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (5255.4033ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (6440.2701ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (4831.1721ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (5220.1231ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (4752.4774ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (5266.8698ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (5336.4765ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (5190.6959ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (5412.2879ms)\n  ✔ parallel\\test-async-local-storage-bind.js (5214.4809ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (4986.4912ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (5119.2678ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (5009.0531ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (5631.6422ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (5229.3409ms)\n▶ AsyncContextFrame (6581.1624ms)\nℹ tests 23\nℹ suites 1\nℹ pass 22\nℹ fail 1\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 6603.5486\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-async-functions.js (4540.0772ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-async-functions.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 2)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":7152.335,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}}]}